setup.sh

```diff
@@ -0,0 +1,2 @@
+pwsh ./scripts/build.ps1 -DisableLog -DisablePerf -Parallel 12
```

Comment:
Please remove this file.

Reply:
These are really draft to share current status...